src/providers/ConfigCodeActionProvider.ts

```diff
@@ -0,0 +1,135 @@
+import * as vsc from 'vscode';
+import {
+  getStatsigAPICheckGateRegex,
+  getStatsigAPIGetConfigRegex,
+  isLanguageSupported,
+  SupportedLanguageType,
+} from '../lib/languageUtils';
+import { DiagnosticCode } from './diagnostics';
+
+type CodeActionType = {
+  command: string;
+  title: string;
+  kind: vsc.CodeActionKind;
+};
+
+export const CODE_ACTIONS: Record<string, CodeActionType> = {
+  removeStaleConfig: {
+    command: 'remove-stale-config',
+    title: 'Cleanup',
+    kind: vsc.CodeActionKind.QuickFix,
+  },
```

Comment:
I would make two actions for gates: replace with `true` or replace with `false`

Reply:
Considering these are for gates that they probably forgot about, I feel like it might be more user-friendly to just not give them the option. What if we just replace with `false`?

Reply:
Is it possible a user could forget about a gate that always returns true, such as, after a rollout has completed? In which case `false` isn't ideal.